purchase_analytic/models/purchase.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+# © 2016  Laetitia Gangloff, Acsone SA/NV (http://www.acsone.eu)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class PurchaseOrder(models.Model):
+    _inherit = 'purchase.order'
+
+    project_id2 = fields.Many2one(
```

Comment:
maybe it would be better to call it `manual_project_id`?

Reply:
From my side it is fine to keep the field name like this.